.kitura-test.sh

```diff
@@ -10,11 +12,16 @@ if [ $SWIFT_TEST_STATUS -ne 0 ]; then
   return $SWIFT_TEST_STATUS
 fi
 
+# For now, short-circuit kitura tests until those are stabalized.
+return 0
```

Comment:
I've create issue #249 to remember that...

Reply:
I'd suggest to add a `// TODO: ` statement as well. Most IDEs collect and aggregate them. I for example use CLion, and this way I won't lose track of unkept promises.